gap/ProsetDerivedMethods.gi

```diff
@@ -4,6 +4,17 @@
 # Implementations
 #
 
+## FIXME: CategoryFilter := C -> HasIsThinCategory( C ) and IsThinCategory( C ) and CanCompute( C, "UniqueMorphism" )
```

Comment:
What's the background of this `FIXME`, i.e. why would you like to check for ´UniqueMorphism`?

Reply:
For a first approximation see https://github.com/homalg-project/Locales/issues/71.

There are two sorts of thin categories:
* The morphism has void/empty/trivial MorphismDatum
* The morphism has a MorphismDatum which witnesses it being a morphism

In the first case, such thin categories should be able to define `UniqueMorphism` as the CAP operation mentioned in the above-mentioned issue.

Reply:
I see. But why should `UniqueMorphism` not be installed in the second case (if there is an algorithm for computing such a witness from given source and range)?

Reply:
Yes, this `UniqueMorphism` should be called `UniqueMorphismWithoutGivenWitness` or the other one should be called `UniqueMorphismWithGivenWitness`. We are open for suggestions.